src/extension.ts

```diff
@@ -21,8 +21,15 @@ export function activate(context: vscode.ExtensionContext) {
 				return;
 			}
 
-			// Not a symlink. No need to open real path.
 			if (filePath === realFilePath) {
+				// Not a symlink. No need to open real path.
+				lastRealFilePath = realFilePath;
+				return;
+			}
+			if (lastRealFilePath === realFilePath) {
```

Comment:
I tested this on my Windows machine and the file paths do not match. There's something up with the casing with these two variables causing them to be different.

Reply:
I'm fine merging this in without handling this. I'll just make a windows specific gh issue. But if you have an idea I'm all ears!

I tried `fs.realpath.native` to see if that made a difference, but it caused some very strange behaviour that I cannot debug at the moment.

Reply:
Yeah, I have an idea: issue `fs.stat` additionally and compare tuples `(stats.ino, stats.rdev)`
Trying to write it now in the most async typescript way possible 😄

Reply:
Please check if it works for you now.

Reply:
no dice :[

lets revert the fs stat change and I'll add a gh issue detailing the problem for Windows. That way macOS and linux users can still benefit.

Reply:
I first tested another version that compared stat fields one by one, then force pushed slightly better version that seems to be broken because you cannot compare arrays with `===` in TypeScript. Sorry, my first time writing it 😅

Reply:
Please try once more
Now I checked that it works with Windows symlinks, albeit opened from WSL.

Reply:
the go back is working!